### DISCUSSION TITLE
Want new features? Add your wish to the wishlist!

### DISCUSSION THREAD

**Discussion by YoruCathy** [empriselab/RCareWorld#7] [Ideas / Suggestions]
Greetings 👋! As we keep working on the project, we would like more suggestions from the users. Here is a wishlist you can put the features you want in. Feel free to put the functionalities you want in this thread.

**Comment by kavikode** [comment]
Please include a package for navigation of the robots and a simple navigation control for the robots.

**Comment by DanielTakeshi** [comment]
@YoruCathy has the Notion page been changed or updated since the post? I tried accessing the documentation but I get the shown access error. 

<img width="513" alt="Screenshot 2023-11-12 at 2 12 19 PM" src="https://github.com/empriselab/RCareWorld/assets/5532096/9bf5c798-927c-4500-bb06-40dbe914a29e">

**Reply by YoruCathy** [reply]
I realized notion links are easily damaged if I move the page somewhere else so you can just reply to this thread for the functionalities you want.

**Reply by DanielTakeshi** [reply]
Oh wait, this Notion is working!
https://sunny-building-1e7.notion.site/Full-install-RCareWorld-All-you-need-to-know-c5432d938f9545299f2b268373ec8abc

Basically my wish list was to get some documentation to figure out what commands to run, etc. :) 

And also to get a file like this which I see was added last week:
https://github.com/empriselab/RCareWorld/blob/main/CONTRIBUTING.md

**Reply by DanielTakeshi** [reply]
@YoruCathy may you check this part?

<img width="742" alt="Screen Shot 2023-11-21 at 10 44 49 AM" src="https://github.com/empriselab/RCareWorld/assets/5532096/ec375cb8-0f19-4623-91d6-57fec73ea7fd">

**Comment by DuinoDu** [comment]
Is that possible to open source Unity project? So I can add new environment in the RCareWorld.
